spotinst/ocean_aks_virtual_node_group_auto_scaling/fields_spotinst_ocean_aks_virtual_node_group_auto_scaling.go

```diff
@@ -137,7 +161,7 @@ func flattenAutoScale(autoScale *azure.VirtualNodeGroupAutoScale) []interface{}
 }
 
 func expandHeadrooms(data interface{}, nullify bool) ([]*azure.VirtualNodeGroupHeadroom, error) {
-	list := data.([]interface{})
+	list := data.(*schema.Set).List()
```

Comment:
need to make sure it is not a breaking change for users that already have `autoscale_headroom` in their configuration